tests/tests_suite_test.go

```diff
@@ -279,9 +281,18 @@ func (s *existingSspStrategy) Init() {
 }
 
 func (s *existingSspStrategy) Cleanup() {
-	if s.ssp != nil {
-		s.RevertToOriginalSspCr()
+	if s.ssp == nil {
+		return
 	}
+
+	s.RevertToOriginalSspCr()
+
+	// Waiting for PVCs that are imported by DataImportCrons to be in bound state.
+	//
+	// The tests in dataSources_test.go remove and recreate DataImportCrons, and
+	// they can cause the PVCs to be removed. To keep the cluster in healthy state after tests,
+	// we wait until PVCs are reimported.
+	s.waitUntilPVCsAreBound()
```

Comment:
I believe PVC/Snapshot knowledge is a little bit overkill for SSP.
Maybe just wait for DataImportCron to be UpToDate==true?

Reply:
done